Sources/App/Controllers/API/TodoController.swift

```diff
@@ -0,0 +1,33 @@
+import Vapor
+import FluentSQLite
+
+/// Controls basic CRUD operations on `Todo`s.
+final class TodoController {
```

Comment:
This should conform to `RouteCollection` via: https://github.com/vapor-community/styleguide#routes-and-controllers